testflinger.env

```diff
@@ -6,3 +6,5 @@
 #MONGODB_PASSWORD=testflinger
 #MONGODB_DATABASE=testflinger_db
 #MONGODB_ROOT_PASSWORD=testflinger
+# If you specify this variable the others are unneeded/ignored.
+#MONOGDB_URI=mongodb://mongo:27017/testflinger_db
```

Comment:
There's a typo. It should be "MON**GO**DB", not "MON**OG**DB"

Reply:
Ah, thx for catching, will update.